com.avaloq.tools.ddk.check.ui/src/com/avaloq/tools/ddk/check/ui/builder/util/CheckMarkerHelpExtensionHelper.java

```diff
@@ -260,17 +260,7 @@ private boolean isCatalogContextId(final QualifiedName catalog, final URI uri, f
    * @return the check type
    */
   private String getCheckType(final Check check) {
-    TriggerKind trigger = check.getKind();
-    switch (trigger) {
-    case FAST:
-      return CHECKTYPE_PRAEFIX + ".fast";
-    case NORMAL:
-      return CHECKTYPE_PRAEFIX + ".normal";
-    case EXPENSIVE:
-      return CHECKTYPE_PRAEFIX + ".expensive";
-    default:
-      return null;
-    }
+    return CHECKTYPE_PREFIX + '.' + check.getKind().toString().toUpperCase(Locale.ENGLISH);

```

Comment:
To preserve the original functionality, should this be the following?
```suggestion
    return CHECKTYPE_PREFIX + '.' + check.getKind().toString().toLowerCase(Locale.ENGLISH);
```

Reply:
done